_pytest/config.py

```diff
@@ -104,7 +104,10 @@ def _prepareconfig(args=None, plugins=None):
     elif not isinstance(args, (tuple, list)):
         if not isinstance(args, str):
             raise ValueError("not a string or argument list: %r" % (args,))
-        args = shlex.split(args)
+        if sys.platform == "win32":
```

Comment:
@RonnyPfannschmidt of course meant `shlex.split(args, posix=sys.platform != 'win32')`. :wink: